index.bs

```diff
@@ -472,8 +470,54 @@ template contents). It consistes of these steps:
             |configuration|["{{SanitizerConfig/dataAttributes}}"] is true
       1. If |handleJavascriptNavigationUrls| and &laquo;[|elementName|, |attrName|]&raquo; matches an entry in the
          [=built-in navigating URL attributes list=], and if |attribute|'s [=protocol=] is
-         "`javascript:`":
-         1. Then remove |attribute| from |child|.
+         "`javascript:`",
+         then [=/remove=] |attribute| from |child|.
+      1. If |handleJavascriptNavigationUrls| and |child|'s [=Element/namespace=]
+         [=string/is=] the [=MathML Namespace=] and |attr|'s
+         [=Attr/local name=] [=string/is=]
+         "`href`" and  |attr|'s [=Attr/namespace=] [=string/is=] `null` or the
```

Comment:
Seems weird to use string/is on null. Just use "is" without reference?

Reply:
Done.